src/util/userUrlHandling.ts

```diff
@@ -54,9 +56,23 @@ export const getFileIdFromUrl = (
     url: string,
     idParam?: string | string[] | null
 ) => {
-    // currently only id we're extracting from the url, but possible we'll need more
     if (isGoogleDriveUrl(url)) {
         return getGoogleDriveFileId(url, idParam);
+    } else {
+        // ex) "mysite.com/myTraj.simularium?dl=0" -> "myTraj.simularium"
+        const urlSplit = url.split("/");
+        return urlSplit[urlSplit.length - 1].split("?")[0];
+    }
+};
+
+export const getRedirectUrl = (url: string, fileName: string | undefined) => {
+    if (url && fileName && USER_TRAJ_REDIRECTS.includes(url)) {
```

Comment:
I know this is for this particular set of redirects, and we might never need this again.... but if we are trying to make this useful for the future, a redirect mapping makes more sense to me.... instead of assuming the filename is going to be consistent between the user trajectory url and the final networked file name. Ie `return USER_TRAJ_REDIRECTS_MAPPING[url]` which would just return the value of the key:value mapping.

Reply:
I thought about that, but then I got to thinking about how to deal with dev/staging/prod environments/URLs and decided to go with this. But then, we only really need the redirects for prod, right? If that's the case I can just go ahead and make a mapping, that would be very simple! 👍🏻 👎🏻 ?

Reply:
Ahh, I tried to refactor this and things got more complicated than I thought, so I'll merge this as is. If we ever need to add more redirects I think we can think about making this more general-purpose.